src/TestFramework/TestFramework/Attributes/TestMethod/ConditionBaseAttribute.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+public enum Mode
+{
+    Include,
```

Comment:
Xml doc: 
Include: runs only when the condition is met.
Exclude: skips when the condition is met.